logstore/logstore.go

```diff
@@ -234,6 +239,13 @@ func (ls *logstore) getLog(id thread.ID, lid peer.ID) (info thread.LogInfo, err
 	if err != nil {
 		return
 	}
+	direct, err := ls.GetBool(id, lid.Pretty())
```

Comment:
This is maybe not as 'safe' as having a dedicated Direct flag API (because we're just using the log id as the generic key here). I could be convinced to create a more bespoke API for the Direct flag if that seems more appropriate?

Reply:
What about making the key `<logID>/direct` or similar?